guides/common/modules/con_upgrade-path-overview.adoc

```diff
@@ -0,0 +1,39 @@
+[id="upgrade-path-overview"]
+= Upgrade path overview
+
+Upgrading to {ProjectName} {ProjectVersion} from {ProjectName} {ProjectVersionPrevious} includes the following high-level steps:
+
+ifdef::foreman-deb,foreman-el,orcharhino[]
+. Upgrading your {ProjectServer} to {ProjectVersion}.
+. Upgrading your {SmartProxyServers} to {ProjectVersion}.
+endif::[]
+ifdef::katello,satellite[]
+. Ensuring that your {ProjectServer}s and {SmartProxyServers} are running on {Project} {ProjectVersionPrevious}.
+. Upgrading your {ProjectServer}:
+.. Upgrading your {ProjectServer} to {ProjectVersion}.
+.. Synchronizing the new {ProjectVersion} repositories.
+. Upgrading your {SmartProxyServers} to {ProjectVersion}.
+endif::[]
+
+ifndef::orcharhino[]
+You can upgrade your {SmartProxies} separately over multiple maintenance windows because {SmartProxies} at version {ProjectVersionPrevious} and {ProjectVersionPrevious-Previous} will keep working with your upgraded {ProjectServer} {ProjectVersion}.
```

Comment:
```suggestion
You can upgrade your {SmartProxies} separately over multiple maintenance windows because versions {ProjectVersionPrevious} and {ProjectVersionPrevious-Previous} remain compatible with your upgraded {ProjectServer} {ProjectVersion}.
```